bundles/af-core/pom.xml

```diff
@@ -464,20 +464,19 @@
         -->
 
         <dependency>
-            <groupId>com.adobe.aem</groupId>
-            <artifactId>aem-forms-sdk-api</artifactId>
-        </dependency>
-
-
-        <dependency>
-            <groupId>com.adobe.aem</groupId>
-            <artifactId>aem-sdk-api</artifactId>
+            <groupId>com.adobe.aemfd</groupId>
+            <artifactId>aemfd-client-sdk</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
```

Comment:
reason behind this?

Reply:
aemfd client SDK brings some old transitive dependencies which we don't need